frontend/package.json

```diff
@@ -6,6 +6,7 @@
     "@bcgov/bc-sans": "1.0.1",
     "@react-keycloak/web": "1.0.0-alpha.6",
     "@reduxjs/toolkit": "1.2.3",
+    "@types/react-paginate": "6.2.1",
```

Comment:
^ The type reference should be a **devDependency** as it only provides intellisense and type support for TypeScript. Just a minor tweak that can be dealt as tech-debt later